setup-static-pods.sh

```
@@ -0,0 +1,414 @@
+    sudo kubebuilder/bin/kubectl create configmap kube-root-ca.crt --from-file=ca.crt=/tmp/ca.crt -n default 2>/dev/null || true
+
+    if ! is_running "kubelet"; then
+        echo "Starting kubelet..."
+        sudo PATH=$PATH:/opt/cni/bin:/usr/sbin kubebuilder/bin/kubelet \
+            --kubeconfig=/var/lib/kubelet/kubeconfig \
+            --config=/var/lib/kubelet/config.yaml \
+            --root-dir=/var/lib/kubelet \
+            --cert-dir=/var/lib/kubelet/pki \
+            --tls-cert-file=/var/lib/kubelet/pki/kubelet.crt \
+            --tls-private-key-file=/var/lib/kubelet/pki/kubelet.key \
+            --hostname-override=$(hostname) \
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)